src/Maestro/tests/Maestro.ScenarioTests/MaestroScenarioTestBase.cs

```diff
@@ -169,13 +233,61 @@ public async Task<AsyncDisposableValue<string>> CreateSubscriptionAsync(string s
                 return AsyncDisposableValue.Create(subscriptionId, async () =>
                 {
                     TestContext.WriteLine($"Cleaning up Test Subscription {subscriptionId}");
-                    await RunDarcAsync("delete-subscriptions", "--id", subscriptionId, "--quiet").ConfigureAwait(false);
+                    try
+                    {
+                        await RunDarcAsync("delete-subscriptions", "--id", subscriptionId, "--quiet").ConfigureAwait(false);
+                    }
+                    catch (MaestroTestException)
+                    {
+                        // If this throws an exception the most likely cause is that the subscription was deleted as part of the test case
+                    }
                 });
             }
 
             throw new MaestroTestException("Unable to create subscription.");
         }
 
+        public async Task<string> GetSubscriptionInfo(string subscriptionId)
+        {
+            return await RunDarcAsync("get-subscriptions", "--ids", subscriptionId).ConfigureAwait(false);
+        }
+
+        public async Task<string> GetSubscriptions(string channelName)
+        {
+            return await RunDarcAsync("get-subscriptions", "--channel", $"\"{channelName}\"");
```

Comment:
We shouldn't have to have these quotes here... this implies RunDarcAsync isn't quoting parameters correctly itself.